src/eventer/eventer_jobq.c

```diff
@@ -875,12 +876,23 @@ void eventer_jobq_set_max_backlog(eventer_jobq_t *jobq, uint32_t max) {
   jobq->max_backlog = max;
 }
 
+void eventer_jobq_set_floor(eventer_jobq_t *jobq, uint32_t floor_concurrency) {
+  if(floor_concurrency > jobq->min_concurrency)
+  jobq->min_concurrency = floor_concurrency;
```

Comment:
I don't see how min and floor are different.  Floor modifies min here.  And min modifies floor in set_min_max

Reply:
floor <= min <= desired <= max


Floor is the minimum number of threads when idle.

Desired is the max number of threads when there is a job backlog.

Desired can only be changed within [min, max]

Reply:
ok, that makes sense.